packages/atomic/src/components/search/result-template-components/atomic-quickview-modal/atomic-quickview-modal.tsx

```diff
@@ -142,6 +169,27 @@ export class AtomicQuickviewModal implements InitializableComponent {
     return !!this.content && !!this.result;
   }
 
+  private handleKeywordHighlight() {
```

Comment:
This method name seems redundant with the above `watchHighlightKeywords`.

I think this method should be split in two: `enableHighlights` / `disableHighlights`.